src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -109,6 +110,7 @@
               :state             CertificateState
               :dns_alt_names     [schema/Str]
               :subject_alt_names [schema/Str]
+              :authorization_extensions {schema/Str schema/Str}
```

Comment:
if we're already cleaning up indentation.. might wanna realign everything with this 😅

Reply:
We decided to stop doing aligning maps like that. It causes too much whitespace churn each time a key is added.

The indentation I fixed is what was needed for my IDE to do fancy parenthesis things correctly.

Reply:
ah okay, that's fair!